back/src/test/java/com/cocktailpick/back/user/docs/UserDocumentation.java

```diff
@@ -25,4 +25,10 @@ public static RestDocumentationResultHandler findMe() {
 			)
 		);
 	}
+
+	public static RestDocumentationResultHandler deleteMe() {
+		return document("user/me",
```

Comment:
```suggestion
		return document("user/delete",
```
위의 `findMe()`와 `deleteMe()` 메서드의 문서가 모두 generated-snippets 디렉토리 아래의 `user/me` 로 똑같은 파일에 만들어지고 있어요. 그래서 위의 `findMe()`의 파일명을 `user/find`로 바꾸고, `deleteMe()` 파일명도 `user/delete`와 같이 변경을 해줘야 정상적으로 문서가 만들어질 것 같아요. :)